tools/checkformat/checkformat_config.xml

```diff
@@ -0,0 +1,222 @@
+| Parameter : num + label    |  Format |
++============================+=========+
+| 1 : Data matrix file       | tabular |
++----------------------------+---------+
+| 2 : Sample metadata file   | tabular |
++----------------------------+---------+
+| 3 : Variable metadata file | tabular |
++----------------------------+---------+
+
+| The **required formats** for the dataMatrix, sampleMetadata, and variableMetadata files are described in the **HowTo** entitled 'Format Data For Postprocessing' available on the main page of Workflow4Metabolomics.org (https://nextcloud.inrae.fr/s/qLkNZRf84QQ5YLY).
+
+
+----------
+Parameters
+----------
+    
+Data matrix file
+  | variable x sample **dataMatrix** tabular separated file of the numeric data matrix, with . as decimal, and NA for missing values; the table must not contain metadata apart from row and column names; the row and column names must be identical to the rownames of the sample and variable metadata, respectively (see below)
+  |
+
+Sample metadata file
+  | sample x metadata **sampleMetadata** tabular separated file of the numeric and/or character sample metadata, with . as decimal and NA for missing values
+  |
+  
+Variable metadata file
+  | variable x metadata **variableMetadata** tabular separated file of the numeric and/or character variable metadata, with . as decimal and NA for missing values
+  |
+
+Make syntactically valid sample and variable names
+  | if set to 'yes', sample and variable names will converted to syntactically valid names with the 'make.names' R function when required (e.g. an 'X' is added to names starting with a digit, blanks will be converted to '.', etc.)
+  |
+  
+------------
+Output files
+------------
+
+dataMatrix_out.tabular
+  | dataMatrix data file; may be identical to the input dataMatrix in case no renaming of sample/variable names nor re-ordering of samples/variables (see the 'information' file for the presence/absence of modifications)
+  |
+
+sampleMetadata_out.tabular
+  | sampleMetadata data file; may be identical to the input sampleMetadata in case no renaming of sample names nor re-ordering of samples (see the 'information' file for the presence/absence of modifications)
+  |
+
+variableMetadata_out.tabular
+  | variableMetadata data file; may be identical to the input variableMetadata in case no renaming of variable names nor re-ordering of variables (see the 'information' file for the presence/absence of modifications)
+  |
+
+information.txt
+  | Text file with all messages when error(s) in formats are detected
+  |
+    
+---------------------------------------------------
+    
+---------------
+Working example
+---------------
+    
+.. class:: infomark
+  
+See the **W4M00001a_sacurine-subset-statistics**, **W4M00001b_sacurine-complete**, **W4M00002_mtbls2**, or **W4M00003_diaplasma** shared histories in the **Shared Data/Published Histories** menu.
+    
+---------------------------------------------------
+    
+----
+NEWS
```

Comment:
I would skip the version from the tool help. It will be in the TS which is linked from the tool form